pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -338,4 +340,9 @@
      * @return consumer builder.
      */
     ConsumerBuilder<T> intercept(ConsumerInterceptor<T> ...interceptors);
+
+    /**
+     * Set dead letter policy for consumer
```

Comment:
Please expand the comment, explaining what are the purpose and scope of dead-letter policy (for people who are not familiar with the concept). 
Also it would be nice to provide here a quick code example that explain how to enable this feature.

Reply:
ok